src/components/Reports/PeopleReport/PeopleReport.jsx

```diff
@@ -460,24 +449,38 @@ class PeopleReport extends Component {
         isActive={isActive}
         name={`${firstName} ${lastName}`}
       >
+        <p>{userProfile.role}</p>
+
+        {userProfile.endDate ? (
+          <p>
+            <Checkbox
+              value={this.state.isRehireable}
+              onChange={() => this.setRehireable(!this.state.isRehireable)}
+              label="Rehireable"
+            ></Checkbox>
+          </p>
```

Comment:
Warning: validateDOMNesting(...): &lt;div&gt; cannot appear as a descendant of &lt;p&gt;.
```suggestion
          <div>
            <Checkbox
              value={this.state.isRehireable}
              onChange={() => this.setRehireable(!this.state.isRehireable)}
              label="Rehireable"
            ></Checkbox>
          </div>
```